### ISSUE TITLE
How to change expired TLS certificate in Eclipse Che

### ISSUE THREAD

**Issue by tathagata1977** [eclipse/che#22877]
### Summary

I am running Eclipse Che for a year now. Time has come to change TLS certificates. 
In configmap the cert are embedded what I observed. 
So what I am expecting changing the secret in my case its (chedev) might not be good enough. 
Please suggest 
Thanks

### Relevant information

CHE_INFRA_KUBERNETES_TLS__CERT: |+ the embedded certificate are in configmap che.
and also secret file is there!

**Comment by tathagata1977** [comment]
Any One know this ?

Thanks

**Comment by tolusha** [comment]
Hello
Please have a look https://github.com/eclipse/che/issues/21534#issuecomment-1177316426

**Comment by tathagata1977** [comment]
Thanks @tolusha ,
For my case what I am observing is # 
my  tls is chedev                         kubernetes.io/tls which is expired, and I am trying to add a new one. 
1st I try to update chedev secret , but its not validating. 
then I observed configmap che having those tls cert embedded as well, probably in time of installation its configured. 
I used chectl server:deploy --installer=operator --platform=k8s --multiuser --chenamespace=dev --domain=dev.cataneo.de --k8spodreadytimeout=300000 --k8spodwaittimeout=3000000 --debug --che-operator-cr-patch-yaml=base-config.yaml to install eclipse che. 
Thanks
